pkg/nsx/services/common/types.go

```diff
@@ -156,3 +156,17 @@ type VPCResourceInfo struct {
 	ID       string
 	ParentID string
 }
+
+type VPCNetworkConfigInfo struct {
+	IsDefault               bool
```

Comment:
does VPCNetworkConfigInfo need to move here? or just keep it in vpc/types.go? if we won't hit the cycle refer issue

Reply:
I moved here per Yun's comment in another patch which is one vpc_dev. it makes sense, because this struct is used for both VPC service and NamespaceController, so it is a common resource.